checkstyle/suppressions.xml

```diff
@@ -540,6 +540,8 @@
     <suppress checks="ClassFanOutComplexityCheck" files="com.hazelcast.map.impl.proxy.MapProxySupport"/>
     <suppress checks="ClassDataAbstractionCoupling" files="com.hazelcast.map.impl.proxy.MapProxySupport"/>
     <suppress checks="CyclomaticComplexity" files="com.hazelcast.map.impl.client.AbstractTxnMapRequest"/>
+    <suppress checks="MethodCount|ClassDataAbstractionCoupling" files="com.hazelcast.map.impl.MapServiceContextImpl"/>
+    <suppress checks="MethodCount" files="com.hazelcast.map.impl.MapServiceContext"/>
```

Comment:
The problem is that the complexity was there, but it was hidden. To deal with too many methods and data coupling issues, super classes and super interfaces were introduced. 

In case of the MapServiceContext is only made the whole thing a lot more complex because you now need to deal with a lot more twice the number of structures and half of them they can't carry their weight.

So the simplification is that these unwanted interface/abstract-class folded into the place where they should have been. And yes.. now you get the checkstyle issue back.

The next step is to figure out how to correctly reduce complexity.. and not by applying just some cosmetic changes.